src/web/layouts/Standard.stories.tsx

```diff
@@ -25,7 +25,7 @@ mockRESTCalls();
 export default {
     title: 'Layouts/Standard',
     parameters: {
-        chromatic: { viewports: [1300], delay: 800 },
+        chromatic: { viewports: [1300], delay: 800, diffThreshold: 0.2 },
```

Comment:
Is this related to this PR?

Reply:
Not directly. But I needed to use _a_ PR to see if this change fixed this problem which has been occuring recently